test/e2e/common/node/container_probe.go

```diff
@@ -562,6 +562,59 @@ var _ = SIGDescribe("Probing container", func() {
 		pod := gRPCServerPodSpec(nil, livenessProbe, "etcd")
 		RunLivenessTest(f, pod, 1, defaultObservationTimeout)
 	})
+
+	/*
+		Testname: Stop probing a pod during graceful shutdown
+		Description: When a pod's deletionTimestamp is set (indicating graceful shutdown initiated) to allow a quiet shutdown by setting liveness probe results success before disabling probing.
```

Comment:
The description doesn't quite match the actual test... what you are testing is that the container doesn't become unhealthy (indicating a livenessProbe failure) once the pod deletion has been requested (graceful shutdown indeed).